src/main/java/io/xpring/xrpl/model/XRPPathElement.java

```diff
@@ -58,10 +55,25 @@
    * PathElement protocol buffer</a>
    */
   static XRPPathElement from(PathElement pathElement) {
+    Optional<String> account = Optional.empty();
+    if (pathElement.hasAccount()) {
+      account = Optional.of(pathElement.getAccount().getAddress());
```

Comment:
is it possible that `.getAddress()` will return `null`?  If so, you should change to 
```java
Optional.ofNullable(pathElement.getAccount().getAddress());
```

You have this control logic in a bunch of different places, so I'd suggest making that change everywhere

Reply:
So the reason I didn't use this type of logic is because the Java protobufs don't return `null`... so what would instead happen is we'll get an `Optional` object that has some sort of value such as an empty string, when what we really want is an `Optional` that is truly empty and so can be handled as such.

Reply:
Ah right, I forgot protobufs have default empty values.

So what will happen here is that if `pathElement.getAccount().getAddress()` returns `""`, then `Optional.of` that will not produce an empty `Optional`.  Maybe I'm missing something but it should probably be 
`Optional.ofNullable(pathElement.getAccount().getAddress().isEmpty() ? null : pathElement.getAccount().getAddress())`